VetClinic.DAL/Context/FakeDbContext.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace VetClinic.DAL.Context
+{
+    public class FakeDbContext : DbContext
```

Comment:
Why is it FakeDbContext ?

Reply:
> Why is it FakeDbContext ?

Just needed some sort of stub in order to implement the base interface that uses DbContext